test/expr_test.py

```diff
@@ -4,7 +4,7 @@
 
 def get_pixel_value(clip):
     frame = clip.get_frame(0)
-    arr = frame[0]
+    arr = frame.get_read_array(0)
```

Comment:
As per my comment, https://github.com/vapoursynth/vapoursynth/pull/738#pullrequestreview-738117742, I still see no reason to apply this revert commit.